addons/website_livechat/models/im_livechat_request.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields
+
+
+class LivechatRequest(models.Model):
+    _name = 'im_livechat.request'
```

Comment:
Could be great to replace by a mail.channel where livechat_active=True, livechat_visitor_id=current_vid; with indexes should be quick and therefore no need to have another model.